ote/src/cljs/ote/views/admin/detected_changes.cljs

```diff
@@ -17,29 +17,65 @@
             [ote.style.admin :as style-admin]
             [cljs-time.core :as t]
             [ote.ui.form :as form]
-            [ote.ui.tabs :as tabs]))
+            [ote.ui.tabs :as tabs]
+            [ote.style.buttons :as button-styles]))
 
 (defn detect-changes [e! app-state]
   [:div
    [:div (stylefy/use-style (style-base/flex-container "column"))
+    [:div (stylefy/use-style style-admin/button-container)
```

Comment:
Couldn't here view use some of nap's already existing generic buttons which use right layout and style automagically?
`ote.ui.buttons`: `save`, `cancel`, `delete`, out of which `save` is the primary-button which should be used here.